portal/templates/layouts/courseInfo.html

```diff
@@ -0,0 +1,26 @@
+
+{% extends 'layouts/base.html' %}
+
+{% block content %}
+<section class='edit'>
+
+  <header>
+    <h1>{{ course['course_title']}}</h1>
+    <a href="/courseManagement">Back</a>
+    <!--<a class='action' href="{{ url_for(courseInfo.update), id=course['course_num']}}">Edit</a>-->
```

Comment:
why is this line commented it out? should it be deleted or inserted?

Reply:
courseInfo.html should not exist. It is not part of the user story.